pittAPI.py

```diff
@@ -412,9 +413,9 @@ def get_person(self, query, maxPeople=10):
 
                 if personurl.lower().startswith("https://") and hasattr(ssl, '_create_unverified_context'):
                     ct = ssl._create_unverified_context()
-                    f = urlopen(personurl, context=ct)
+                    f = s.get(personurl, context=ct)
```

Comment:
In requests, there is no `context` arg for the `get()` method. This results in an error. Please look up the proper way to disable SSL verification in requests!